pages/agent/v3/cli_env.md

```diff
@@ -1,6 +1,6 @@
 # `buildkite-agent env`
 
-The Buildkite Agent's `env` subcommands provide the ability to inspect environment variables.
+The Buildkite Agent's `env` subcommands provide the ability to inspect environment variables. In order for jobs to inspect and modify the environment variables the agent needs to have expermiment flag set with job-api like `--experiment=job-api`
 
 With the job-api experiment enabled, jobs can inspect and modify their environment variables using the `get`, `set`, and `unset` sub-commands. These provide an alternative to using shell commands to inspect and modify environment variables.
```

Comment:
Since this paragraph already talks about the job-api experiment, how about we add it here instead?

```suggestion
With the job-api experiment enabled, jobs can inspect and modify their environment variables using the `get`, `set`, and `unset` sub-commands. These provide an alternative to using shell commands to inspect and modify environment variables. To enable the job-api experiment, include the `--experiment=job-api` flag when starting the agents.
```

Is it right to say "when starting the agents"?

Reply:
@mbelton-buildkite Yes when starting the agents is the right thing to say